src/components/snackbar/snackbar.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { useState, forwardRef, useImperativeHandle } from "react"
+import "./snackbar.css"
+
+export const Snackbar = forwardRef((props, ref) => {
```

Comment:
nit naming: this is usually referred to as a `Toast`

Reply:
That's what I get learning from a Canadian